checker/build.gradle

```diff
@@ -176,6 +200,11 @@ task allNullnessTests(type: Test, group: 'Verification') {
     include '**/Nullness*.class'
 }
 
+task allCalledMethodsTests(type: Test, group: 'Verification') {
```

Comment:
Why do we need this new task?

Reply:
This is a convenience task for local testing, because this checker has multiple test configurations (i.e. with different sets of framework support enabled, or with the `-AuseValueChecker` option enabled, etc.). The Nullness Checker has a similar task, on which this one is modeled.